app/controllers/web/home.php

```diff
@@ -425,11 +427,11 @@
 
         switch ($format) {
             case 'swagger2':
-                $format = new Swagger2($utopia, $services, $routes, $models, $keys[$platform], $authCounts[$platform] ?? 0);
+                $format = new Swagger2($utopia, $services, $routes, $models, $keys[$platform], $authCounts[$platform] ?? 0, $allModels);
```

Comment:
I think this is a leftover from the initial commits?

Reply:
👍🏻